docs/user/project_metadata.md

```diff
@@ -30,21 +30,44 @@ PyPI currently supports several ways of verifying project URLs. When a URL is ve
     PyPI package owner, and does not imply any additional safety about that
     URL or any other relationship to the project in question.
 
-### Self-links
+The following subsections specify the different types of URLs that can be verified.
 
-Any URL to the project on PyPI will be considered verified.
+#### Self-links
 
-### Via Trusted Publishing
+Any URL to the project on PyPI will be considered verified. For example, the project page for `pip` will mark all of the
+following as verified:
+
+- https://pypi.org/project/pip/
+- https://pypi.org/p/pip/
+- https://pypi.python.org/project/pip
+- https://pypi.python.org/p/pip
+- https://python.org/pypi/pip
+
+#### Via Trusted Publishing
 
 [Trusted Publishing](trusted-publishers/index.md) allows PyPI to attest that the 
 publishing workflow for a package is coming from a verified source.
 
-PyPI supports the following OIDC Publishers:
+The URLs that can be verified depend on the Trusted Publisher used:
+
+- [GitHub Actions][gh-action-tab]: Packages uploaded using GHA from a repository will have the GitHub URLs for that
+  repository verified. For example, for the `pypa/pip` repository, the following URLs will be verified:
+    - https://github.com/pypa/pip
+    - https://github.com/pypa/pip/* (all subpaths)
+    - https://github.com/pypa/pip.git
+    - https://pypa.github.io/pip
+    - https://pypa.github.io/pip/* (all subpaths)
+- [GitLab CI/CD][gitlab-tab]: Packages uploaded using GitLab CI/CD from a repository will have the GitLab URLs for that
+  repository verified. For example, for the `pypa/pip` repository, the following URLs will be verified:
+    - https://gitlab.com/pypa/pip
+    - https://gitlab.com/pypa/pip/* (all subpaths)
+    - https://gitlab.com/pypa/pip.git
+- [Google Cloud][gc-tab]: No Google-specific URLs are currently verified.
+- [ActiveState][active-tab]: Packages uploaded using ActiveState will have URLs linking to the project in ActiveState 
```

Comment:
```suggestion
- [ActiveState][active-tab]: Packages uploaded using ActiveState will have URLs linked to the project in ActiveState 
```

Reply:
NITPICK: Is there any reason not to reuse the `pypi/pip` example for ActiveState?

Reply:
nope, fixed!

Reply:
For the first suggestion, I'm not sure if "having URLs linked to" is better than "having URLs linking to". I'll leave the question open for the native English speakers

Reply:
I think both are idiomatic, but "linked" is probably what I would have written. I'll accept this suggestion 🙂